extensions/smoke-test/manifest.json

```diff
@@ -0,0 +1,1232 @@
+{
+  "version": 1,
+  "locale": "en_US",
+  "platform": "4.4.2",
+  "metadata": {
+    "appmode": "quarto-static",
+    "primary_rmd": "index.qmd",
+    "primary_html": null,
+    "content_category": null,
+    "has_parameters": false
+  },
+  "environment": {
+    "r": {
+      "requires": "~=4.4.0"
+    }
+  },
```

Comment:
We should add Python and Quarto versions to the `environment` dict so that we can have more flexible requirements there (e.g. what's the minimum Python version this will run on? `~3.12`? I'm not sure what we've been trying to target with extensions, maybe @dotNomad knows).

I tried restoring the _renv_ environment on R 4.5 and MASS failed to install, so not sure how a restore on a Connect server that lacked 4.4 but had 4.5 would go. 🤔

Reply:
This is generally a challenge with gallery content right now.

Reply:
- Python 3.11 is the minimum the content will support.
- a test deployment of the content succeeded against R 4.5